src/init-message.ts

```diff
@@ -1,10 +1,11 @@
+import { LaunchApplication } from "./publishable";
 
 export type Identifier       = string | number;
 export type ProtocolVersion  = string;
-export type ISODateString = string;
-export const InitMessageName = "SharinatorInit";
-
+export type ISODateString    = string;
 
+export const InitMessageName = "SharinatorInit";
```

Comment:
It would help me in the codap-shim app if the messages kept a common prefix *and* that prefix was exported from init-message.  I have a nearly blind proxy that pumps everything but the Sharinator messages.  Something like this:

```
export const MessagePrefix = "Sharinator"
export const InitMessageName = `${MessagePrefix}Init`
export const InitResponseMessageName = `${MessagePrefix}InitResponse`
```